src/chapters/data-and-variables/variables.rst

```diff
@@ -1,14 +1,236 @@
+
+.. sourcecode:: js
+
+    "JavaScript" = programmingLanguage;
+
+.. tip::
+
+   To avoid confusion when reading or writing code, say to yourself 
+   
+        ``programmingLanguage`` is assigned ``'JavaScript'``
+
+   or 
+    
+        ``programmingLanguage`` gets the value ``'JavaScript'``. 
+    
+   Don't say 
+    
+        ``programmingLanguage`` equals ``'JavaScript'``.
+
+.. index:: 
+   pair: variable; global
+
+.. warning:: What if, by mistake, you leave off ``let`` when declaring a variable?
+
+   .. sourcecode:: js
+
+      programmingLanguage = "JavaScript";
+
+   Contrary to what you might expect, JavaScript will not complain or throw an error. In fact, creating a variable without ``let`` is valid syntax, but is results in very different behavior. Such a variable will be a **global variable**, which we will learn more about in a later lesson. The main point to keep in mind for now is that you should always use ``let`` unless you have a specific reason not to do so.
+
+Evaluating Variables
+--------------------
+
+.. index:: variable; evaluation
+
+After a variable has been created, it may be used later in a program in any place where a value may be used. For example, we know that ``console.log`` prints a value, so we can also give ``console.log`` a variable.
+
+.. sourcecode:: js
+
+   console.log("Hello, World!");
+
+.. sourcecode:: js
+
+   let message = "Hello, World!";
+   console.log(message);
+
+These two examples have the exact same same output.
+
+Using a variable in this way is called **evaluating** a variable. If you ask JavaScript to evaluate a variable, it will produce the value that is currently linked to the variable. The effect is as if the value of the variable is substituted for the variable name in the code when executed.
+
+Think of 
+
+.. sourcecode:: js
+
+    let message = "What's up, Doc?";
+    let n = 17;
+    let pi = 3.14159;
+
+    console.log(message);
+    console.log(n);
+    console.log(pi);
+
+**Output**
+
+::
+
+    What's up, Doc?
+    17
+    3.14159
+
+In each case, the printed result is the value of the variable. 
+
+Like values, variables also have types. We determine the type of a variable the same way we determine the type of a value, using ``typeof``.
+
+.. sourcecode:: js
+    
+    let message = "What's up, Doc?";
+    let n = 17;
+    let pi = 3.14159;
+
+    console.log(typeof message);
+    console.log(typeof n);
+    console.log(typeof pi);
+
+**Output**
+
+::
+
+   string
+   number
+   number
+
+The type of a variable is the type of the data it currently refers to.
+
+Reassigning Variables
+---------------------
+
+We use variables in a program to "remember" things, like the current score at the football game. But variables are *variable*. This means they can change over time, just like the scoreboard at a football game. You can assign a value to a variable, and later assign it a different value.
```

Comment:
Please change the sentence "But variables are variable". It is confusingly worded and unnecessary.